### DISCUSSION TITLE
When I use `blink.cmp` extra, some plugins become ineffective

### DISCUSSION THREAD

**Discussion by gengdz** [LazyVim/LazyVim#4700] [Q&A]
When I use `blink.cmp` extra, the following plugins become ineffective. 

![image](https://github.com/user-attachments/assets/c87301b3-476f-4c06-846c-a37f538d18a6)

I think that `codeium.nvim` and `tailwindcss-colorizer-cmp.nvim` should not become ineffective; there should be some compatibility code for `blink.cmp` extra.

**Comment by folke** [comment]
tailwind-colorizer-cmp isn't really needed with blink, since blink will colorize the kind icon and name as the tailwind color.

As for AI plugins, I tried the blink-compat code for copilot, but that didn't work unfortunately, so in the copilot case, I've changed it so that it now completes the suggestion on `<tab>`, while blink completes on `<CR>`.

Still need to update the other AI plugins indeed.

Added an issue https://github.com/LazyVim/LazyVim/issues/4702

**Reply by KoinBanks** [reply]
Hi @folke Any idea why copilot chat is not working right now when using blink in extras? I get this error:

```
...hare/nvim/lazy/CopilotChat.nvim/lua/CopilotChat/init.lua:339: in function <...hare/nvim/lazy/CopilotChat.nvim/lua/CopilotChat/init.lua:338>
E5108: Error executing lua: ...hare/nvim/lazy/CopilotChat.nvim/lua/CopilotChat/init.lua:339: attempt to index field 'chat' (a nil value)
```

It works when I don't have blink enabled in extras. 

Dotfiles without blink enabled (nothing special here): [https://github.com/iFarmGolems/nvim](https://github.com/iFarmGolems/nvim)

NVIM v0.10.2
Distro is Arch WSL

**Reply by dpetka2001** [reply]
See https://github.com/LazyVim/LazyVim/discussions/4711#discussioncomment-11153133
